src/server/server.go

```diff
@@ -63,6 +69,32 @@ func PprofMem(w http.ResponseWriter, r *http.Request) {
         }
 }
 
+func PprofCpuStart(w http.ResponseWriter, r *http.Request) {
+	log.Printf("Starting cpu profiling\n")
+	w.Header().Add("Content-Type", "application/octet-stream")
+
+	if err := pprof.StartCPUProfile(w); err != nil {
+	    log.Fatal("could not start CPU profile: ", err)
+	}
+
+	select {
+	case <-cpuProfTimeout:
```

Comment:
Does this end point need to block until profiling is completed, or can we just send back a message immediately to indicate that profiling has started?